pkg/migrations/op_create_constraint.go

```diff
@@ -189,13 +188,11 @@ func (o *OpCreateConstraint) Rollback(ctx context.Context, conn db.DB, s *schema
 func (o *OpCreateConstraint) removeTriggers(ctx context.Context, conn db.DB) error {
 	dropFuncs := make([]string, len(o.Columns)*2)
 	for i, j := 0, 0; i < len(o.Columns); i, j = i+1, j+2 {
-		dropFuncs[j] = pq.QuoteIdentifier(TriggerFunctionName(o.Table, o.Columns[i]))
-		dropFuncs[j+1] = pq.QuoteIdentifier(TriggerFunctionName(o.Table, TemporaryName(o.Columns[i])))
+		dropFuncs[j] = TriggerFunctionName(o.Table, o.Columns[i])
+		dropFuncs[j+1] = TriggerFunctionName(o.Table, TemporaryName(o.Columns[i]))
 	}
-	_, err := conn.ExecContext(ctx, fmt.Sprintf("DROP FUNCTION IF EXISTS %s CASCADE",
-		strings.Join(dropFuncs, ", "),
-	))
-	return err
+	dropFunctions := NewDropFunctionAction(conn, dropFuncs...)
+	return dropFunctions.Execute(ctx)
 }
```

Comment:
This function is quite hard to read. Could we take this opportunity to simplify it to something like:

```go
func (o *OpCreateConstraint) removeTriggers(ctx context.Context, conn db.DB) error {
	dropFuncs := make([]string, 0, len(o.Columns)*2)
	
	for _, column := range o.Columns {
		dropFuncs = append(dropFuncs, pq.QuoteIdentifier(TriggerFunctionName(o.Table, column)))
		dropFuncs = append(dropFuncs, pq.QuoteIdentifier(TriggerFunctionName(o.Table, TemporaryName(column))))
	}
	
  dropFunctions := NewDropFunctionAction(conn, dropFuncs...)
	return dropFunctions.Execute(ctx)
}
```